src/python/pants/backend/python/util_rules/pex_cli.py

```diff
@@ -12,67 +12,22 @@
 from pants.backend.python.subsystems.python_native_code import PythonNativeCodeSubsystem
 from pants.backend.python.subsystems.setup import PythonSetup
 from pants.backend.python.util_rules import pex_environment
+from pants.backend.python.util_rules.pex_cli_tool import PexCli, PexPEX
+from pants.backend.python.util_rules.pex_cli_tool import rules as pex_cli_tools_rules
 from pants.backend.python.util_rules.pex_environment import PexEnvironment, PexSubsystem
-from pants.core.goals.resolves import ExportableTool
-from pants.core.util_rules import adhoc_binaries, external_tool
+from pants.core.util_rules import adhoc_binaries
 from pants.core.util_rules.adhoc_binaries import PythonBuildStandaloneBinary
-from pants.core.util_rules.external_tool import (
-    DownloadedExternalTool,
-    ExternalToolRequest,
-    TemplatedExternalTool,
-)
 from pants.engine.fs import CreateDigest, Digest, Directory, MergeDigests
 from pants.engine.internals.selectors import MultiGet
-from pants.engine.platform import Platform
 from pants.engine.process import Process, ProcessCacheScope
 from pants.engine.rules import Get, collect_rules, rule
-from pants.engine.unions import UnionRule
 from pants.option.global_options import GlobalOptions, ca_certs_path_to_file_content
-from pants.option.option_types import ArgsListOption
 from pants.util.frozendict import FrozenDict
 from pants.util.logging import LogLevel
-from pants.util.meta import classproperty
-from pants.util.strutil import softwrap
 
 logger = logging.getLogger(__name__)
 
 
-class PexCli(TemplatedExternalTool):
```

Comment:
Moved this class to `pex_cli_tool.py` so that plugin resolution could depend directly on the Pex download rules.